### ISSUE TITLE
When opening a legacy Wikipedia file (e.g. 0.5), the info on how to read it disappears after a few seconds

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js-pwa#557]
The info merely says "switch to jQuery mode", but it's easy to miss because of auto-dismiss code.

I either need to prevent auto-dismiss from dismissing the info for such types, or else auto-switch the user to jQuery mode and pop up a dialogue box explaining why. If we auto-switch, it should be temporary until the next ZIM archive is loaded. That shouldn't be too complicated.
